NEWS.rst

```diff
@@ -0,0 +1,958 @@
+.. comment
+    This is the changelog file for DFHack.  If you add or change anything, note
+    it here under the heading "DFHack Future", in the appropriate section.
+    Items within each section are listed in alphabetical order to minimise merge
+    conflicts.  Try to match the style and level of detail of the other entries.
+
+    Sections for each release are added as required, and consist solely of the
+    following in order as subheadings::
+
+        Internals
+        Lua
+        New [Internal Commands | Plugins | Scripts | Tweaks]
+        Fixes
+        Misc Improvements
+        Removed
+
+    When referring to a script, plugin, or command, use backticks (```) to
+    create a link to the relevant documentation - and check that the docs are
+    still up to date!
+
+
+#########
+Changelog
+#########
+
+.. contents::
+   :depth: 2
+
+
+DFHack Future
+=============
+
+Internals
+---------
+- A method for caching screen output is now available to Lua (and C++)
+- Developer plugins can be ignored on startup by setting the DFHACK_NO_DEV_PLUGINS environment variable
+- The console on Linux and OS X now recognizes keyboard input between prompts
+- JSON libraries available (C++ and Lua)
+- More DFHack build information used in plugin version checks and available to plugins and lua scripts
+- Fixed a rare overflow issue that could cause crashes on Linux and OS X
+- Stopped DF window from receiving input when unfocused on OS X
+- Fixed issues with keybindings involving Ctrl-A and Ctrl-Z, as well as Alt-E/U/N on OS X
+- Multiple contexts can now be specified when adding keybindings
+- Keybindings can now use F10-F12 and 0-9
+- Plugin system is no longer restricted to plugins that exist on startup
+- :file:`dfhack.init` file locations significantly generalized
+
+Lua
+---
+- Scripts can be enabled with the built-in enable/disable commands
+- A new function, reqscript(), is available as a safer alternative to script_environment()
+- Lua viewscreens can choose not to intercept the OPTIONS keybinding
+
+New internal commands
+---------------------
+- kill-lua: Interrupt running Lua scripts
+- type: Show where a command is implemented
+
+New plugins
+-----------
+- `confirm`: Adds confirmation dialogs for several potentially dangerous actions
+- `fix-unit-occupancy`: Fixes issues with unit occupancy, such as faulty "unit blocking tile" messages (:bug:`3499`)
+- title-version (formerly vshook): Display DFHack version on title screen
+
+New scripts
+-----------
+- `burial`:  sets all unowned coffins to allow burial ("-pets" to allow pets too)
```

Comment:
Do the single backticks serve a purpose? They seem to be ignored by Github's renderer.